### PR TITLE
Update copyright format

### PR DESCRIPTION
**Pull request by elharo**
@chingor13

### CODE REVIEW

**Comment by chingor13** [comment]
Note, this will cause the tests to fail until we convert all the headers

**Comment by elharo** [comment]
In that case what do you think about disabling the check since it isn't smart enough to distinguish between new and modified code?

File: java.header [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ^/\*$
-^ \* Copyright \d\d\d\d, Google Inc\. All rights reserved\.$
+^ \* Copyright \d\d\d\d, Google LLC$
```
Comment (chingor13) [inline java.header]: This looks like a regex - we can try:
```suggestion
^ \* Copyright \d\d\d\d, Google (Inc\. All rights reserved\.|LLC)$
```

File: java.header [hunk 2]
```diff
@@ -12,7 +12,7 @@
 ^ \* in the documentation and/or other materials provided with the$
 ^ \* distribution\.$
 ^ \*$
-^ \*    \* Neither the name of Google Inc\. nor the names of its$
+^ \*    \* Neither the name of Google LLC nor the names of its$
```
Comment (chingor13) [inline java.header]:
```suggestion
^ \*    \* Neither the name of Google (Inc\.|LLC) nor the names of its$
```

**Comment by chingor13** [comment]
@elharo The java.header uses regex, so the suggested fixes will work and keep backwards compatibility.

**Pull request merged**